src/app/(homepage)/Intro.js

```diff
@@ -5,28 +5,30 @@ import Typography from "@mui/material/Typography";
 import useMediaQuery from "@mui/material/useMediaQuery";
 import { useTheme } from "@mui/material/styles";
 import Grid from "@mui/material/Grid";
+import { useRouter } from "next/navigation";
 
 function IntroText(props) {
+  const router = useRouter()
```

Comment:
demo.png is very big, 3000x3000, should be included with Next Image component, otherwise will be loaded long